godot-core/src/registry/callbacks.rs

```diff
@@ -186,21 +186,24 @@ pub unsafe extern "C" fn default_get_virtual<T: UserClass>(
     T::__default_virtual_call(method_name.as_str())
 }
 
+/// # Safety
+///
+/// - `r_is_valid` must be assignable.
+/// - `r_is_valid` comes uninitialized and must be set.
 pub unsafe extern "C" fn to_string<T: cap::GodotToString>(
     instance: sys::GDExtensionClassInstancePtr,
-    is_valid: *mut sys::GDExtensionBool,
+    r_is_valid: *mut sys::GDExtensionBool,
```

Comment:
Please revert the rename. While it follows Godot convention, it's inconsistent with all the other callback signatures here. It's even inconsistent with the other parameter in this exact function (would be `p_instance` in Godot).